src/Akeneo/Pim/Automation/DataQualityInsights/front/src/application/component/ProductEditForm/QualityScoreProductHeader.tsx

```diff
@@ -1,12 +1,12 @@
 import React from 'react';
 import {QualityScoreBar} from '../QualityScoreBar';
 import {useCatalogContext, useFetchProductQualityScore} from '../../../infrastructure/hooks';
+import {QualityScoreLoader} from '../QualityScoreLoader';
 
 const QualityScoreProductHeader = () => {
   const {channel, locale} = useCatalogContext();
-  const score = useFetchProductQualityScore(channel, locale);
-
-  return <QualityScoreBar currentScore={score ? score : null} />;
+  const {score, isLoading} = useFetchProductQualityScore(channel, locale);
+  return isLoading ? <QualityScoreLoader /> : <QualityScoreBar currentScore={score ? score : null} />;
```

Comment:
```suggestion
  return isLoading ? <QualityScoreLoader /> : <QualityScoreBar currentScore={score} />;
```
because score is either null either a score valid value

Reply:
Done